src/ucs/stats/stats.c

```diff
@@ -77,7 +77,7 @@ static ucs_stats_context_t ucs_stats_context = {
     .root_node        = {},
     .root_filter_node = {},
     .lock             = PTHREAD_MUTEX_INITIALIZER,
-    .thread           = 0xfffffffful
+    .thread           = (pthread_t)-1
```

Comment:
why don't use `UCS_SPINLOCK_OWNER_NULL` here?
what if we decide to change `#define UCS_SPINLOCK_OWNER_NULL ((pthread_t)0)`
we should just include <ucs/type/spinlock.h> file in the file

Reply:
it's not related to spinlock - it holds the stats thread id

Reply:
* `src/ucs/stats/stats.c` doesn't use `spinlock`.  It's the member of `ucs_stats_context_t`, Should I use `UCS_SPINLOCK_OWNER_NULL`?
* Should I change deafult value to `zero`?

```C
typedef struct {
    volatile unsigned    flags;

    ucs_time_t           start_time;
    ucs_stats_filter_node_t  root_filter_node;
    ucs_stats_node_t     root_node;
    ucs_stats_counter_t  root_counters[UCS_ROOT_STATS_LAST];

    union {
        FILE             *stream;         /* Output stream */
        ucs_stats_client_h client;       /* UDP client */
    };

    union {
        int              signo;
        double           interval;
    };

    khash_t(ucs_stats_cls) cls;

    pthread_mutex_t      lock;
    pthread_t            thread;
} ucs_stats_context_t;
```

Reply:
@yosefe, @dmitrygx. Previous my and @yosefe's comment posted at the same time.
Please let me know if I need to change the default value. And Could you run CI?

Reply:
> it's not related to spinlock - it holds the stats thread id

@yosefe @hiroyuki-sato thank you! got it